sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/hybridsearch/FullTextQueryStatistics.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.query.hybridsearch;
+
+import com.azure.cosmos.implementation.Constants;
+import com.azure.cosmos.implementation.JsonSerializable;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class FullTextQueryStatistics {
+    @JsonProperty(Constants.Properties.HIT_COUNTS)
+    private List<Long> hitCounts;
+    @JsonProperty(Constants.Properties.TOTAL_WORD_COUNT)
+    private Long totalWordCount;
+    private JsonSerializable jsonSerializable;
```

Comment:
Why do you need the JsonSerializable here?

Reply:
Removed it.